### ISSUE TITLE
constructor Builder in class Builder cannot be applied to given types; required: Context found: UploadService,String reason: actual and formal argument lists differ in length

### ISSUE THREAD

**Issue by Harin-Kaklotar** [gotev/android-upload-service#341]
I just simply download the project and add as module in my project. I set support version 26+ and initialize very well. 
but when I run the project it stop on compile time and gave me this error 
`Error:(540, 55) error: constructor Builder in class Builder cannot be applied to given types;
required: Context
found: UploadService,String
reason: actual and formal argument lists differ in length` 
for UploadTask.java class

here is the screen shot of class 

![notification_error](https://user-images.githubusercontent.com/9987008/33884877-77e5a108-df67-11e7-86fa-16a80b7bb622.png)

this error accrue in three methods

1.  createNotification
2.  updateNotificationProgress
3.  updateNotification

so how to fix that and run the project ???

**Comment by Harin-Kaklotar** [comment]
@gotev yes are right 👍  there are gradle issue. after fix, it works perfectly :)
and thanks for this awesome library

**Comment by gotev** [comment]
That's great :smiley: and thank you!
